src/fu-security-attr.c

```diff
@@ -207,6 +207,314 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_BOOTGUARD_VERIFIED) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("BootGuard prevents unauthorized device software from operating when the "
+			 "device is started.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_BOOTGUARD_POLICY) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "Intel BootGuard Error Policy ensures the device does not continue to start if "
+		    "its device software has been tampered with.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_CET_ENABLED) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_CET_ACTIVE) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Intel Control-Flow Enforcement Technology detects and prevents certain "
+			 "methods for running malicious software on the device.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_SMAP) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Intel Supervisor Mode Access Prevention prevents critical parts of "
+			 "device memory from being accessed by less secure programs.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_ENCRYPTED_RAM) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "Encrypted RAM makes it impossible for information that is stored in device "
+		    "memory to be read, if the memory chip is removed and accessed.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_IOMMU) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("IOMMU protection prevents connected devices from accessing unauthorized "
+			 "parts of system memory.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_LOCKDOWN) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Linux Kernel Lockdown mode prevents administrator (root) accounts from "
+			 "accessing and changing critical parts of system software.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_TAINTED) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "Linux Kernel Verification makes sure that critical system software has "
+		    "not been  tampered with. Using device drivers which are not provided with the "
+		    "system can  revent this security feature from working correctly.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_SWAP) == 0) {
+		/* TRANSLATORS: longer description */
+		return _("Kernel Swap temporarily saves information to disk as you work. This "
+			 "information which could be accessed by someone if they gained access to "
+			 "the disk.");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_RAM) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_IDLE) == 0) {
+		/* TRANSLATORS: longer description */
+		return _(
+		    "During system sleep the device’s memory chip could be removed, allowing the "
```

Comment:
Since the same is used for S3/S2I you probably need to make it clear which is which

Reply:
The designers wanted to make the title the unique thing, and make the description more generic.

Reply:
Maybe a second sentence? `Using Suspend-to-Idle is better than Suspend to Disk` etc.

Reply:
Yeah something like that would work.

Reply:
Done.